### PR TITLE
Add GNOME Theme

### PR DESCRIPTION
**Pull request by mechtifs**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **

My package is a theme to match GNOME's latest adwaita style.

There are no packages like it in Package Control.

### CODE REVIEW

File: repository/g.json [hunk 1]
```diff
@@ -2334,6 +2334,16 @@
 					"branch": "master"
 				}
 			]
+		},
+		{
+			"name": "GNOME Theme",
+			"details": "https://github.com/mechtifs/sublime-gnome-theme",
+			"releases": [
+				{
+					"sublime_text": "*",
```
[the remark this answers is not in the document]
Comment (braver) [reply repository/g.json:2343]: Yeah, we don't really care about that though, that's really ancient.

File: repository/g.json [hunk 2]
```diff
@@ -2334,6 +2334,16 @@
 					"branch": "master"
 				}
 			]
+		},
+		{
+			"name": "GNOME Theme",
+			"details": "https://github.com/mechtifs/sublime-gnome-theme",
+			"releases": [
```
Comment (braver) [inline repository/g.json:2341]: Your package is both a theme and a color scheme. It's probably a good idea to explain as much, and to add some labels.

**Comment by braver** [comment]
What is "Gnome" about this? Any relation to the Linux desktop environment?

Anyway, please respond to the feedback to continue the review.

**Comment by mechtifs** [comment]
Sorry for the late reply!!!! Recently busy working on schoolwork and didn't see the comments😢
> What is "Gnome" about this? Any relation to the Linux desktop environment?
Exactly referring to the GNOME desktop environment. This theme is intended to make sublime text look more like native GNOME apps.

**Comment by braver** [comment]
Alright, that's cool. Please let us know when you've handled jrappen's feedback and added the correct labels to your package control entry.

**Comment by braver** [comment]
I'm marking it as stale now. Please let us know when you're done handling the feedback.

**Comment by braver** [comment]
Closing due to lack of activity. We can always re-open an continue the review when you're ready for it.

**Pull request closed without merging**